tests/try_alloc.rs

```diff
@@ -91,6 +91,36 @@ fn main() {
         };
     }
 
+    fn test_static_size_alloc(assert_alloc_ok: fn(bump: &Bump), assert_alloc_err: fn(bump: &Bump)) {
+        // Unlike with `try_alloc_layout`, it's not that easy to test a variety
+        // of size/capacity combinations here.
+        // Since nothing in Bump is really random, and we have to start fresh
+        // each time, just checking each case once is enough.
+        for &fail_alloc in &[false, true] {
+            let bump = GLOBAL_ALLOCATOR.with_successful_allocs(|| {
+                // We can't query the remaining space in the current chunk,
+                // so we have to create a new Bump for each test and fill it to
+                // the brink of a new allocation.
+                let bump = Bump::try_new().unwrap();
```

Comment:
Sorry I missed this before, but the `chunk_capacity` method gives this information, so we shouldn't need to create a whole new bump every time.

Reply:
I think there might be slightly ambiguous wording between my comment and the documentation here:

```rust
    /// Gets the remaining capacity in the current chunk (in bytes).
    ///
    /// […]
    pub fn chunk_capacity(&self) -> usize {
        let current_footer = self.current_chunk_footer.get();
        let current_footer = unsafe { current_footer.as_ref() };

        current_footer as *const _ as usize - current_footer.data.as_ptr() as usize
    }
```

`chunk_capacity` returns the total capacity that remains after allocating the header (as expected by its name, in my opinion), but I'd need the live value of remaining unallocated space to do this reliably.

It might still be possible to work with that when making assumptions about chunk/`ChunkFooter` and `u8` alignments, but that doesn't feel like a clean solution...

Since the code is hit only eight times during testing, I'd like to leave it as-is for simplicity's sake.
Actually exposing the remaining free space in a chunk wouldn't be useful, I think, since it can't be used to predict whether a given layout would allocate.

I'll adjust the comment to make it unambiguous, though.